src/Umbraco.Web/HealthCheck/Checks/Config/TrySkipIisCustomErrorsCheck.cs

```diff
@@ -38,7 +38,7 @@ public override IEnumerable<AcceptableConfiguration> Values
         {
             get
             {
-                var recommendedValue = _serverVersion >= new Version("7.5.0")
+                var recommendedValue = _serverVersion >= new Version("7.5")
```

Comment:
Ah hmm, well, I don't really like that it's kind of "magical" that 7.5.0 is greater than 7.5 but yeah, my correction would also be wrong. So your server version is `7.5` without the `.0`?

Sure, but then the comment in the translations is also wrong. :)
You only need a `%0%` and a `%1%` for this specific message. No need to pass in a variable and then not use it, that's just confusing (as we can see now).